webapp/src/main/webapp/src/app/groups/delete-group.modal.ts

```diff
@@ -0,0 +1,29 @@
+import { BsModalRef } from "ngx-bootstrap";
+import { Component } from "@angular/core";
+import { Group } from "./model/group.model";
+import { GroupService } from "./groups.service";
+
+@Component({
+  selector: 'set-active-modal',
+  templateUrl: './delete-group.modal.html'
+})
+export class DeleteGroupModalComponent {
+  group: Group;
+  unableToDelete: boolean = false;
+
+  constructor(private bsModalRef: BsModalRef, private service: GroupService) {
+  }
+
+  cancel() {
+    this.bsModalRef.hide();
+  }
+
+  save() {
+    this.group.isDeleted = true;
+    this.service.update(this.group).subscribe(res => {
```

Comment:
I would think here that "update" is an implementation detail of soft delete and we would actually want to call this service method delete no?